.setup/bin/setup_sample_courses.py

```diff
@@ -909,6 +909,10 @@ def create(self):
                         print("Inserting {} for {}...".format(gradeable.id, user.id))
                         ins = gradeable_data.insert().values(g_id=gradeable.id, gd_user_id=user.id,
                                                              gd_overall_comment="lorem ipsum lodar")
+                        if gradeable.grade_released_date < NOW and random.random() < 0.5:
+                            ins = gradeable_data.insert().values(g_id=gradeable.id, gd_user_id=user.id,
+                                                             gd_overall_comment="lorem ipsum lodar",
+                                                             gd_user_viewed_date=NOW.strftime("%Y-%m-%d %H:%M:%S%z"))
```

Comment:
change this block to be:
```
values = {'g_id': gradeable.id, 'gd_user_id': user.id, 'gd_overall_comment': 'lorem ipsum lodar'}
if gradeable.grade_released_date < NOW and random.random() < 0.5:
    values['gd_user_viewed_date'] = NOW.strftime('%Y-%m-%d %H:%M:%S%z')
ins = gradeable_data.insert().values(**values)
```
This uses [unpacking argument lists](https://docs.python.org/3/tutorial/controlflow.html#unpacking-argument-lists) to convert the dictionary to the keyword format for the function.